### PR TITLE
say "Save done" only when it's actually done

### PR DESCRIPTION
**Pull request by toyboot4e**
`leaf.el` + `leaf-manager.el` = awesome 🌟  experience

### CODE REVIEW

File: leaf-manager.el [hunk 1]
```diff
@@ -400,7 +400,8 @@ If FORCE is non-nil, write file if file exist."
                  (yes-or-no-p (format "File exists (%s), overwrite? " leaf-manager-file))))
     (prog1 t
       (with-temp-file leaf-manager-file
-        (insert (leaf-manager--create-contents-string))))))
+        (insert (leaf-manager--create-contents-string))
+        t))))
```
Comment (conao3) [inline leaf-manager.el]: Already this Sexp returns `t` via `prog1`, so last `t` is not needed.

**Review by conao3 — COMMENTED** [review]
Good catch! I made one comment.

File: leaf-manager.el (hunk 1 above)
Comment (toyboot4e) [reply leaf-manager.el]: Reverted this change :)

**Comment by conao3** [comment]
Test failed, but this PR is good! merge.

**Pull request merged**